pkg/cluster/bootstrap.go

```diff
@@ -561,6 +563,10 @@ func (c *Cluster) bootstrap(ctx context.Context) error {
 
 	// bootstrap managed database via HTTPS
 	if c.runtime.HTTPBootstrap {
+		// Assuming we should just compare on managed databases
```

Comment:
I think the concerns are the same for HA SQL clusters, but I guess since in that case we don't have a JoinURL to compare our configuration against, we can't do much about it.

Reply:
That part of the code is pretty unknown to me. Where is the config in HA SQL clusters?